generation/spade_gan/spade_gan.ipynb

```diff
@@ -0,0 +1,2665 @@
+{
```

Comment:
<p>Could you please help remove the output from source and just remaining several lines here, refer to: <a href="https://github.com/Project-MONAI/tutorials/pull/1775/commits/0af874bef5951f069b9d530c138074fc14a514d5" target="_blank">https://github.com/Project-MONAI/tutorials/pull/1775/commits/0af874bef5951f069b9d530c138074fc14a514d5</a></p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>I just re-ran everything making sure things got printed only after 10 epochs (which should end up with one image and 2 prints per 10 epochs). </p>